src/tutorial/runtime/configuration-consolidation/families.rst

```diff
@@ -182,39 +174,6 @@ The ``root`` Family
        [[bar]]
            script = echo $FOO
 
-
-Families and ``cylc graph``
----------------------------
```

Comment:
Could you either keep this but comment it out or add a TODO pointing at the old content with a cylc-doc issue so we remember to come back to this one day.

Reply:
I've put it back commented.